src/Testcontainers/Configurations/Modules/Databases/Neo4jTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace DotNet.Testcontainers.Configurations
+{
+  using System;
+  using System.Linq;
+  using DotNet.Testcontainers.Builders;
+  using JetBrains.Annotations;
+
+  /// <inheritdoc cref="TestcontainerDatabaseConfiguration" />
+  [PublicAPI]
+  public class Neo4jTestcontainerConfiguration : TestcontainerDatabaseConfiguration
+  {
+    private const string Neo4jImage = "neo4j:latest";
```

Comment:
Sorry, looks like I missed that. Can you pin the version please?

Reply:
you got it, thanks!